code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -347,6 +347,10 @@ GLOBAL_VAR(bomb_set)
 /obj/machinery/nuclearbomb/blob_act(obj/structure/blob/B)
 	if(timing == -1.0)
 		return
+	if(timing)	//boom
+		spawn
```

Comment:
spawn(0) is usually used.

Reply:
I copied it directly from the normal nuke explosion code. What's the practical difference between spawn and spawn(0)?

Reply:
Should be the same I believe. Both are done after the proc is done "async". More a styling thing

Reply:
use invoke async. Spawns are extremely discourage these days.

Reply:
Alright. Want me to change it in the process() proc too while I'm there?